portal-kernel/src/com/liferay/exportimport/kernel/lar/PortletDataHandler.java

```diff
@@ -179,6 +179,48 @@ public default String getNamespace() {
 
 	public int getRank();
 
+	/**
+	 * Returns the schema version of the <code>PortletDataHandler</code> and
+	 * generally the whole staging and export/import aspect of a component. An
```

Comment:
@matethurzo This states that the schema version for the data handler **AND** the entire staging aspect of the component are returned. Are these always the same?